lib/query-stream.js

```diff
@@ -29,6 +29,7 @@ module.exports = class QueryStream extends Readable {
     this._pageSize = opts.pageSize != null ? opts.pageSize : 500
     this._backoff = new Backoff({ min: 1e3, max: 300e3 })
     this._sql = sql
+    this._deserialize = opts.deserialize !== false
```

Comment:
@vweevers don't you prefer the default option to be deserialize? and only if someone add an option for disable then it will not deserialize

Reply:
That's what this code does :)

If you don't pass in a `deserialize` option, `this._deserialize` will become `true`.

Reply:
oh my bad, i confused it with the disable option 👍